### DISCUSSION TITLE
EnumPath not generated

### DISCUSSION THREAD

**Discussion by kms0217** [querydsl/querydsl#3486] [Q&A]
I am working on a Kotlin Spring Boot project with the following versions:

Kotlin version: 1.7.22
Spring Boot version: 2.7.7
Querydsl version: 5.0.0
The project is built using Gradle version 7.6.

I added the necessary dependencies to use Querydsl and generated QFiles are created successfully, and most of the fields are generated correctly. However, fields with an enum type are not generated. I have tried rebuilding the project multiple times and also tried resetting the cache, but the issue persists.

I have also tried using both IntelliJ build and run settings and Gradle, and have tried with both nullable and non-nullable enum fields, but the enum fields are still not generated. Can you help me figure out what could be causing this issue?

..
dependencies {
    val querydslVersion = "5.0.0"
    implementation("com.querydsl:querydsl-jpa:$querydslVersion")
    kapt("com.querydsl:querydsl-apt:$querydslVersion:jpa")
    kapt("com.querydsl:querydsl-kotlin-codegen:5.0.0")       //Tried it except for this.
...}

I tried @QueryType(PropertyType.ENUM) 
, @Convet(converter = MyEnumClass::class)

**Comment by lpandzic** [comment]
This might help someone - a possible clue - https://github.com/infobip/infobip-spring-data-querydsl/issues/115#issuecomment-3741565580
